lib/shopify_api/resources/resource_feedback.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class ResourceFeedback < Base
+    class ExistingFeedbackSaved < StandardError; end
+    include DisablePrefixCheck
+
+    STATE_SUCCESS = 'success'
+    STATE_REQUIRES_ACTION = 'requires_action'
```

Comment:
Yeah we don't need to add these constants. We did it thinking it might be a convenience, but it's not going to save much typing, given how long the namespace on each constant would be `ShopifyAPI::ResourceFeedback::STATE_...`

We also added these at a time when there were a lot more states and topics still existed.